### DISCUSSION TITLE
Not getting points after merge pull request.

### DISCUSSION THREAD

**Discussion by vijaychandra1910** [recodehive/awesome-github-profiles#731] [Q&A] [answered]
![Screenshot 2024-10-10 221729](https://github.com/user-attachments/assets/97fcbe95-474c-4857-9300-f9b704819ade)
Not getting points after sending  pull request and merge also. How I get points. I rise issue first then fork it. After solving issue create a new pull request. In this case I also not getting points for solving issue.

**Comment by maniacedoc** [comment, marked as answer]
@vijaychandra1910 The points take some time to be reflected in the leaderboard, if your PR has been merged and it has the appropriate labels then you will get the points after some time.
